Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for Confluent Platform Community Edition
 name: cp-helm-charts
-version: 0.4.1
+version: 0.4.2
```

Comment:
kindly don't update the version here.
I'm doing version change with confluent platform release (which is currently 5.4.1).
Thanks

Reply:
Remove Version config.